assets/src/blocks/Media/MediaFrontend.js

```diff
@@ -19,7 +17,7 @@ const wrapEmbedHTML = embed_html => {
   return wrapperDiv.outerHTML;
 }
 
-export const MediaFrontend = ({ attributes }) => {
+export const MediaFrontend = ( attributes ) => {
```

Comment:
Had to undo this change I suggested earlier on your PR, my bad! It needs to be matched in the frontend rendering code too, but then we would need to change the other frontend components, which is a bit too much for a PR that's already big.